projects/plugins/crm/admin/settings/tax.page.php

```diff
@@ -138,9 +138,29 @@
 					<div class="ui info icon message">
 						<div class="content">
 							<div class="header"><?php esc_html_e( 'No Tax Rates', 'zero-bs-crm' ); ?></div>
-							<p><?php echo wp_kses( sprintf( __( 'There are no tax rates defined yet. Do you want to <a href="%s" id="zbs-new-add-tax-rate">create one</a>?', 'zero-bs-crm' ), '#' ), $zbs->acceptable_restricted_html ); ?></p>
+							<p>
+								<p>
+									<?php
+										echo sprintf(
+											wp_kses(
```

Comment:
I've no idea why the formatting shows as completely off here. Viewing the file it looks fine (as in my code editor, tabs used): https://github.com/Automattic/jetpack/blob/fix/crm-tax-settings-link/projects/plugins/crm/admin/settings/tax.page.php#L144